pipelines/pipeline_katello_32.yml

```diff
@@ -19,9 +19,6 @@
     katello_repositories_version: 3.2
     foreman_repositories_version: 1.13
     katello_repositories_use_koji: true
-    foreman_installer_skip_installer: true
-    foreman_installer_additional_packages:
-      - foreman-installer-katello
```

Comment:
The role I'm proposing replacing this with doesn't have a `foreman_installer_skip_installer` or similar var, I think having a foreman_installer role with with a don't actually run the installer is kind of pointless.

If we want to just install packages, maybe we should extend the `update_os_packages` role or create a `os_packages`.